apps/docs/content/components/forms/TextField.mdx

```diff
@@ -58,7 +58,7 @@ The `isReadOnly` prop makes the TextField's text content immutable. Unlike `isDi
 <Example src="inputs/docs/textField/readOnly" />
 
 ### Error
-A text field with an error message.
+A TextField can be displayed in an error state to indicate that the user input is invalid.
```

Comment:
ok now we switched to "TextField". I'm confused :P

Reply:
Basically what I said above, components who are two words I kept them as is, this might be a mistake. RAC is pretty much all over the place. 

![image](https://github.com/user-attachments/assets/a5b462d5-903f-4860-8eb1-ec727d50e586)

Reply:
ya maybe we should define this... I'm pretty sure I've seen you put "text field" somewhere else.